pkgs/development/tools/misc/babeltrace/common.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchzip
+, autoreconfHook
+, pkg-config
+, glib
+, elfutils
+
+, version
+, hash
+}:
+
+args: stdenv.mkDerivation (finalAttrs: args // {
+  inherit version;
+
+  src = fetchzip {
+    url = "https://www.efficios.com/files/babeltrace/${finalAttrs.pname}-${finalAttrs.version}.tar.bz2";
+    inherit hash;
+  };
+
+  # In 1.x, the pre-generated ./configure script uses an old autoconf version
+  # which breaks cross-compilation (replaces references to malloc with
+  # rpl_malloc).
+  # In 2.x, there is no pre-generated ./configure script.
+  # Re-generate with nixpkgs's autoconf. This requires glib to be present in
+  # nativeBuildInputs for its m4 macros to be present.
+  nativeBuildInputs = args.nativeBuildInputs or [ ] ++ [ autoreconfHook pkg-config glib ];
+
+  buildInputs = args.buildInputs or [ ] ++ [ glib elfutils ];
```

Comment:
Same here - use `extraBuildInputs`.

Reply:
This was actually not used at all so I removed it.